datastore.go

```diff
@@ -110,6 +110,14 @@ type GCDatastore interface {
 	CollectGarbage() error
 }
 
+// PersistentDatastore is an interface that should be implemented by datastores
+// which can report disk usage.
+type PersistentDatastore interface {
+	Datastore
+
+	DiskUsage() (uint64, error)
```

Comment:
Should have a doc explicitly saying what the unit here is (i.e. bytes).

Reply:
I would think that would be obvious, but it can't hurt.

Reply:
yeah you're totally right, my bad